src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/DeliverySessionService.kt

```diff
@@ -265,9 +263,6 @@ class DeliverySessionService(
     additionalInformation: String?
   ): Pair<DeliverySession, Appointment> {
     var sessionAndAppointment = getDeliverySessionAppointmentOrThrowException(referralId, appointmentId)
-    if (sessionAndAppointment.second.appointmentTime.isAfter(OffsetDateTime.now())) {
```

Comment:
This checks are recently added by Neil for fixing the bug where we should not allow to give session feedback to future appointments. I think we need this

Reply:
yes this a regression, we need to keep this

Reply:
reverted